tests/lib/App/AppManagerTest.php

```diff
@@ -680,23 +680,23 @@ public function provideDefaultApps(): array {
 			[
 				'unexist,settings',
 				'files',
-				'{"settings":[1],"files":[2]}',
+				'{"settings":{"app":"settings","order":1},"files":{"app":"files","order":2]}',
```

Comment:
```suggestion
				'{"settings":{"app":"settings","order":1},"files":{"app":"files","order":2}}',
```

Also, where is app key used?

Reply:
In AppManager to figure out the default app for a user. Otherwise you would have to somehow map an ID to a app which is not possible as the ID can be chosen without any restrictions on closures.